library/centos

```diff
@@ -5,11 +5,11 @@ Constraints: !aufs
 
 Tags: latest, centos7, 7
 GitFetch: refs/heads/CentOS-7
-GitCommit: 3463e5f011fbd82f8283757408427802ee020ee4
+GitCommit: 50281d86d6ed5c61975971150adfd0ede86423bb
 
 Tags: centos6, 6
 GitFetch: refs/heads/CentOS-6
-GitCommit: 697f0fae0ff2b942c37eb7f4100163c1e5d3e922
+GitCommit: 5048dc2d18390e22bc9c95fe7526f7f63490b7b6
 
 Tags: centos5, 5
```

Comment:
ah, okay. I was under the impression removing it from the file here would remove it from the hub as well.